sombra-ivan-arabchuk/burger-test-project
# Added login/logout functionality

## Why
Need to add navbar and login/logout functionality
## How 
implemented by creating useAuth custom hook
## Testing
- run app
- see navbar 
- click on the login button
